lib/lockfile.js

```diff
@@ -320,6 +343,8 @@ onExit(() => {
 });
 
 module.exports.lock = lock;
+module.exports.lock.mtimeChecker = createMtimeChecker();
+module.exports.createMtimeChecker = createMtimeChecker;
```

Comment:
This is necessary for the tests right?

Reply:
Yes

Reply:
Actually no need to export when we have a checker for each lock file.

Reply:
Awesome!